app/javascript/components/host-initiator-form/index.jsx

```diff
@@ -32,14 +32,16 @@ const HostInitiatorForm = ({ redirect, storageManagerId }) => {
   };
 
   const onCancel = () => {
-    const message = __('defining of host initiator was cancelled by the user');
+    const message = __('Defining of host initiator was cancelled by the user.');
     miqRedirectBack(message, 'success', redirect);
   };
 
   const validate = (values) => {
     const errors = {};
-    if ((!values.wwpn || !values.wwpn.length) && (!values.custom_wwpn || !values.custom_wwpn.length)) {
-      errors.wwpn = 'Please provide at least one WWPN.';
+    if (values.port_type == "FC" || values.port_type == "NVMeFC") {
```

Comment:
You should include the check for IQN that you added to https://github.com/ManageIQ/manageiq-ui-classic/pull/8399 as well. The only difference I would make would be to change the string to "Please provide at least one IQN" due to the changes made in https://github.com/ManageIQ/manageiq-ui-classic/pull/8398.